<!-- Entry point for using OpenNI devices -->
<launch>

  <arg name="camera" default="sick_visionary_t_driver" />

  <node pkg="sick_visionary_t_driver" type="sick_visionary_t_driver_node" name="$(arg camera)" output="screen" >
    <!-- IP address if the Visionary-T device, default: 192.168.1.10 -->
    <param name="remote_device_ip" value="192.168.97.123" />
    <param name="remote_device_ip_2" value="192.168.97.124" />

    <param name="frame_id" value="camera" />
    <param name="frame_id_2" value="camera_2" />
  </node>

</launch>
